Settings.h

```diff
@@ -104,9 +104,9 @@ typedef struct Settings_ {
 
 #define Settings_cpuId(settings, cpu) ((settings)->countCPUsFromOne ? (cpu)+1 : (cpu))
 
-static inline ProcessField ScreenSettings_getActiveSortKey(const ScreenSettings* this) {
+static inline RowField ScreenSettings_getActiveSortKey(const ScreenSettings* this) {
    return (this->treeView)
```

Comment:
This should only affect sort order if the presented table contains a process list to keep with the spirit of this setting.

Reply:
Hmm?  Any table can have an active sort key (column), its not limited to processes.

Reply:
Ah, I see now - I'll revert it.